app/views/admin/sdg/managers/search.html.erb

```diff
@@ -0,0 +1,32 @@
+<h2><%= t("admin.sdg_managers.search.title") %></h2>
+
+<%= render "admin/shared/user_search", url: search_admin_sdg_managers_path %>
+
+<div id="sdg_managers">
```

Comment:
Do you think it would be worth it to refactor the `index` and the `search` actions so they use only one action and filter by search terms, like it's done in other controllers (I know here we're doing the same thing as is done in other roles)?